<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab1Data</class>
 <widget class="QWidget" name="Tab1Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pPBQuery">
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBServerConnect">
       <property name="text">
        <string>서버연결</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBMenu">
       <property name="text">
        <string>메뉴로 돌아가기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="DBchart">
       <property name="text">
        <string>수신데이터</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,5">
     <item>
      <widget class="QTableWidget" name="pDBTable">
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>57</number>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Detection_Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Locate</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Distance_Level</string>
        </property>
       </column>
       <column/>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
         <item>
          <widget class="QTextEdit" name="pTErecvdata"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="pLErecvid">
         <property name="placeholderText">
          <string>Recipient ID (optional)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pLEsendData"/>
       </item>
       <item>
        <widget class="QPushButton" name="pPBSend">
         <property name="text">
          <string>송신</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
